Assets/Scripts/Managers/ScreenShake/ScreenShakeManager.cs

```diff
@@ -0,0 +1,93 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+public class ScreenShakeManager : MonoBehaviour {
+	private const string _logName = "ScreenShakeManager";
+	public static ScreenShakeManager Instance { get; private set; }
+	Cinemachine.CinemachineVirtualCamera _virtualCamera;
+
+	public AmbientScreenShakeLayer AmbientLayer { get; private set; }
+	public List<ShakeLayer> Layers = new();
+
+	public static float DefaultDuration = .5f;
+	public static Func<float, float> DefaultEasingFunction = Easings.Linear;
+	public static float AmbientShakeAmplitude = 0.5f;
+	public static float AmbientShakeFrequency = 0.1f;
+
+	void Awake() {
+		if (!TryGetComponent(out _virtualCamera)) {
+			Logger.LogError(_logName, _logName);
+		}
+
+		if (Instance == null) {
+			Instance = this;
+		}
+		else {
+			Destroy(gameObject);
+		}
+		DontDestroyOnLoad(gameObject);
```

Comment:
Doesn't do anything because our Vcam isn't a root object.
Breaks the camera if it is.  
ScreenShakeManager gets added to a Vcam with a reference to the player which is lost when transfering scenes, deleting the hypothetical ScreenShakeManager-ed camera present in the new scene. thus losing the player ref.

Reply:
What would be a good alternative?

Reply:
Remove the DontDestoryOnLoad, add it to the camera prefab

Reply:
Did it for you.